packages/material-ui-lab/src/DateTimePicker/DateTimePicker.tsx

```diff
@@ -575,7 +575,7 @@ DateTimePicker.propTypes = {
    * Array of views to show.
    */
   views: PropTypes.arrayOf(
-    PropTypes.oneOf(['date', 'hours', 'minutes', 'month', 'year']).isRequired,
+    PropTypes.oneOf(['date', 'hours', 'minutes', 'seconds', 'month', 'year']).isRequired,
```

Comment:
You could use them in the same order as in the props, i.e. year, date, month, etc...

Reply:
hey Kristiyan96, what's the problem with the order?

Reply:
I assumed it should be either in an increasing or a decreasing order